treeherder/model/migrations/0033_job_detail_field_lengths_decrease.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('model', '0032_bugjobmap_jobnote_models'),
+    ]
+
+    operations = [
+        # Trim the size of the values to meet the new size constraints
+        migrations.RunSQL(
+            ("UPDATE job_detail SET "
```

Comment:
I had thought I might need a where clause here, but I tested this and the update only happens on records where it actually changes the values.  I guess MySQL is smart enough to know when it doesn't actually have to update.  :)